microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/faulttolerance/java/MicroProfileFaultToleranceDiagnosticsParticipant.java

```diff
@@ -135,6 +138,17 @@ private void validateMethod(MethodDeclaration node, List<Diagnostic> diagnostics
 								diagnostics.add(d);
 							}
 						}
+					} else if (isMatchAnnotation(annotation, ASYNCHRONOUS_ANNOTATION)) {
```

Comment:
I will move this code in validateAsynchronousAnnotation (and do the same thing for fallback)